### PR TITLE
Main docs update

### PR DESCRIPTION
**Pull request by Still34**
# Summary

Pushes previous pre-release docs changes to the main repo

### CODE REVIEW

File: docs/_template/light-dark-theme/styles/docfx.vendor.minify.css [hunk 1]
```diff
@@ -361,7 +361,7 @@ pre {
     word-break: break-all;
     word-wrap: break-word;
     color: #333;
-    border: 1px solid #ccc;
+    /*border: 1px solid #ccc;*/
```
Comment (Chris-Johnston) [inline docs/_template/light-dark-theme/styles/docfx.vendor.minify.css]: remove commented out css? I'm surprised this was part of a `minify.css`
```suggestion
```

File: docs/_template/light-dark-theme/styles/gray.css [hunk 2]
```diff
@@ -39,13 +48,13 @@ hr {
 }
 
 /* top navbar */
-.navbar-inverse[role="navigation"] {
+/*.navbar-inverse[role="navigation"] {
```
Comment (Chris-Johnston) [inline docs/_template/light-dark-theme/styles/gray.css:51]: remove this block as well?

File: docs/_template/light-dark-theme/styles/material.css [hunk 3]
```diff
@@ -0,0 +1,201 @@
+body {
+    color: #34393e;
+    line-height: 1.5;
+    /*font-size: 16px;*/
```
Comment (Chris-Johnston) [inline docs/_template/light-dark-theme/styles/material.css:4]:
```suggestion
```
yet another lint

File: docs/_template/light-dark-theme/styles/material.css [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+.navbar-form .form-control {
+    border: none;
+    border-radius: 20px;
+}
+
+/* SIDEBAR */
+
+/*.toc .level1>li {
```
Comment (Chris-Johnston) [inline docs/_template/light-dark-theme/styles/material.css:101]: more commented out style rules

File: docs/_template/light-dark-theme/styles/material.css [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+    border: 2px solid #ddd;
+    border-radius: 20px;
+}
+
+.toc-filter>.filter-icon {
+    display: none;
+}
+
+.sidetoc>.toc {
+    /*background-color: #fff;*/
```
Comment (Chris-Johnston) [inline docs/_template/light-dark-theme/styles/material.css]:
```suggestion
```

File: docs/_template/light-dark-theme/styles/material.css [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+    overflow-x: hidden;
+}
+
+.sidetoc {
+    /*background-color: #fff;*/
```
Comment (Chris-Johnston) [inline docs/_template/light-dark-theme/styles/material.css]:
```suggestion
```

File: src/Discord.Net.Core/Entities/Channels/IMessageChannel.cs [hunk 7]
```diff
@@ -130,12 +121,10 @@ public interface IMessageChannel : IChannel
         ///     of flattening.
         /// </remarks>
         /// <example>
-        ///     The following example downloads 300 messages and gets messages that belong to the user 
-        ///     <c>53905483156684800</c>.
-        ///     <code lang="cs">
-        ///     var messages = await messageChannel.GetMessagesAsync(300).FlattenAsync();
-        ///     var userMessages = messages.Where(x =&gt; x.Author.Id == 53905483156684800);
-        ///     </code>
+        ///     <para>The following example downloads 300 messages and gets messages that belong to the user
```
Comment (Chris-Johnston) [inline src/Discord.Net.Core/Entities/Channels/IMessageChannel.cs:124]:
```suggestion
        ///     <para>The following example downloads 300 messages and gets messages that belong to the user with the ID of
```

File: src/Discord.Net.Examples/Discord.Net.Examples.csproj [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="Core\Entities\Guilds\IGuild.Examples.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Discord.Net.Core\Discord.Net.Core.csproj" />
+    <ProjectReference Include="..\Discord.Net.WebSocket\Discord.Net.WebSocket.csproj" />
+    <PackageReference Include="JetBrains.Annotations" Version="2018.3.0" />
```
Comment (Chris-Johnston) [inline src/Discord.Net.Examples/Discord.Net.Examples.csproj:18]: Not sure if this package was intended?

**Review by Chris-Johnston — CHANGES_REQUESTED** [review]
Just a few nitpicks here and there.

File: src/Discord.Net.Examples/Discord.Net.Examples.csproj (hunk 8 above)
Comment (Still34) [reply src/Discord.Net.Examples/Discord.Net.Examples.csproj:18]: It is to discourage those editing with Rider or R# from cleaning up the methods.

**Pull request merged**